components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/client/IdentityProviderMgtServiceClient.java

```diff
@@ -365,7 +365,8 @@ public Map<String, ProvisioningConnectorConfig> getCustomProvisioningConnectors(
                 for (ProvisioningConnectorConfig config : provisioningConnectorConfigs) {
                     if (!(StringUtils.equals("scim", config.getName()) ||
```

Comment:
we need to change the scim connector to scim1 in the UI level

Reply:
The UI is changed through this change: https://github.com/wso2/carbon-identity-framework/pull/4128/files#diff-1cf495dc14f34c6225ce49ecfe62d3781f12c1a5236f6d59f0e952df12d4ff86L243